NEWS.md

```diff
@@ -4,6 +4,10 @@ MathOptInterface (MOI) release notes
 v0.6.0 (August ??, 2018)
 -----------------------
 
+- Add result fallback for `ConstraintDual` of variable-wise constraint,
+  `ConstraintPrimal` and `ObjectiveValue`.
+- Add tests for `ObjectiveBound`.
```

Comment:
This refers to the PR that hasn't been merged yet, right?

Reply:
No, it refers to https://github.com/JuliaOpt/MathOptInterface.jl/pull/446